src/VirtoCommerce.PricingModule.Data/Handlers/DeletePricesProductChangedEventHandler.cs

```diff
@@ -4,20 +4,22 @@
 using VirtoCommerce.CatalogModule.Core.Events;
 using VirtoCommerce.Platform.Core.Common;
 using VirtoCommerce.Platform.Core.Events;
+using VirtoCommerce.Platform.Core.GenericCrud;
+using VirtoCommerce.PricingModule.Core.Model;
 using VirtoCommerce.PricingModule.Core.Model.Search;
 using VirtoCommerce.PricingModule.Core.Services;
 
 namespace VirtoCommerce.PricingModule.Data.Handlers
 {
     public class DeletePricesProductChangedEventHandler : IEventHandler<ProductChangedEvent>
     {
-        private readonly IPricingService _pricingService;
-        private readonly IPricingSearchService _pricingSearchService;
+        private readonly ICrudService<Price> _priceService;
```

Comment:
IPriceService _priceService
Confusing to see the same name of different types. The _priceService variable should be of the same type in all the module code.